ui/src/app/shared/components/resource-editor/resource.scss

```diff
@@ -1,15 +1,31 @@
 @import 'node_modules/argo-ui/src/styles/config';
 
 .resource {
-  margin: 10px 0;
-  padding: 10px 5px;
-  font: normal 13px/1.2 'Courier New', monospace;
+  padding-top: 1em;
   white-space: pre;
   color: $argo-color-gray-7;
-  border-color: $argo-color-gray-2;;
-  border-width: thin;
-  border-style: inset;
-  overflow-y: scroll;
   width: 100%;
   min-height: 300px;
-}
\ No newline at end of file
+}
+
+.resource-editor-panel {
+  &__editor {
+    border: 1px solid $argo-color-gray-3;
+    border-radius: 5px;
+    margin-top: 1em;
+    padding-bottom: 1em;
+    overflow: hidden;
+  }
+
+  &__lang-toggle {
```

Comment:
can we avoid creating a new style like this - I think the existing "-o" style is acceptable

Reply:
Sounds good. The margin right was to address the inconsistent spacing as shown below: 

<img width="354" alt="Screen Shot 2020-07-23 at 2 33 35 PM" src="https://user-images.githubusercontent.com/6250584/88340917-7fb56380-ccf1-11ea-831f-b56ee57e3373.png">

and the width style was to avoid the button changing sizes each time the user clicks it. Would it be a good idea to inline those instead? I'll remove the `&.yaml` styles.

Reply:
could you use a different (empty?) icon for unchecked?

Reply:
I didn't love how the empty icon for unchecked looked so I used an "x" instead. Does the job